gradle.properties

```
@@ -0,0 +1,5 @@
+group=org.sonarsource.slang
+version=1.16.0-SNAPSHOT
+description=SonarSource Language analyzer
+projectTitle=SLang
```

The logic used for sonar-go was something like:
```suggestion
description=Code Analyzer for Scala
projectTitle=Scala
```
But perhaps it should be `projectTitle=Scala`